project_recalculate/models/project_project.py

```diff
@@ -64,10 +64,9 @@ def project_recalculate(self):
                     project.date):
                 raise Warning(_("Cannot recalculate project because your "
                                 "project don't have date end."))
-            if project.calculation_type != 'none':
-                for task in project.tasks:
-                    task.task_recalculate()
-                vals = project._start_end_dates_prepare()
-                if vals:
-                    project.write(vals)
+            for task in project.tasks:
```

Comment:
Why didn't you add condition here to avoid calculation if ```calculation_type``` is False?

```
if project.calculation_type:
   for task in project.tasks:
      (...)
   (...)
```

Your current code makes no change